src/wms/src/test/java/org/geoserver/wms/wms_1_1_1/LegendCapabilitiesTest.java

```diff
@@ -31,16 +33,22 @@
     // Reusing layer and SLD files from another test; their content doesn't really matter.
     // What is important for this test is the legend info we are adding.
     private static final String LAYER_NAME = "watertemp";
+    private static final String LAYER_NAME_HTP_LEGND = "watertemp_http_legend";
```

Comment:
HTP_LEGND? HTTP_LEGEND was not that much longer (other instance in this file)

Reply:
will rename